src/components/FilterBarInline/FilterBarInline.mdx

```diff
@@ -63,67 +57,84 @@ any Chakra `Box` props, as well as the props below.
+- [MultiSelect](./?path=/docs/components-form-elements-multiselect--docs#accessibility)
+- [Radio](./?path=/docs/components-form-elements-radio--docs#accessibility)
+- [Select](./?path=/docs/components-form-elements-select--docs#accessibility)
+- [Slider](./?path=/docs/components-form-elements-slider--docs#accessibility)
+- [TextInput](./?path=/docs/components-form-elements-textinput--docs#accessibility)
+- [Toggle](./?path=/docs/components-form-elements-toggle--docs#accessibility)
 
 ## FilterBarInline Heading
 
 The `FilterBarInline` component can be rendered with and without a heading. By
 default, when a string is passed to the `heading` prop, the
 `FilterBarInline` component will render an `h2` element sized as a `heading5`
 heading. If this causes accessibility issues in your app, you can set a custom
-heading by passing in a DS `Heading` component.
+heading by passing in a DS `Heading` component in the `heading` prop.
 
-In the following examples, the `FilterBarInline` component will render custom `h3`
-and `h5` elements. Note that no matter what heading level is set, the size will
-always be set to `heading5` internally by the `FilterBarInline` component.
+In the following examples, the `FilterBarInline` component will render custom
+`h3` and `h5` elements. Note that no matter what heading level is set, the size
+will always be set to `heading5` internally by the `FilterBarInline` component.
 
 <Source
   code={`
-
-const customH3 = <Heading level="h3">Custom H3 Heading</Heading>;
-const customH5 = <Heading level="h5">Custom H5 Heading</Heading>;
-
-<FilterBarInline heading={customH3} {...props} />
-<FilterBarInline heading={customH5} {...props} />
-`} language="tsx" />
+  const customH3 = <Heading level="h3">Custom H3 Heading</Heading>;
+  const customH5 = <Heading level="h5">Custom H5 Heading</Heading>;
+  // ...
+  <FilterBarInline heading={customH3} {...props} />
+  <FilterBarInline heading={customH5} {...props} />
+`}
+  language="tsx"
+/>
 
 <Canvas of={FilterBarInlineStories.FilterBarInlineHeading} />
 
 ## Layout Patterns
 
-The display of the `FilterBarInline` child components can be set to a `"row"` or a
-`"column"` layout. On screen width under 600px width the `layout` is
+The display of the `FilterBarInline` child components can be set to a `"row"` or
+a `"column"` layout. On screen width under 600px width the `layout` is
 automatically set to `"column"`.
 
-**Note:** when `layout = "row"` is passed, `MultiSelect` components will have
-`closeOnBlur` set to `true` by default and when `layout = "column"` is passed,
-the `closeOnBlur` prop will be set to `false`.
-
-**Note:** when `layout = "column"` is passed, `MultiSelect` components will have
-`isBlockElement` set to `true` by default and the child components `width`
-will be set to `"full"`.
-
-**Note:** `CheckboxGroup`, `MultiSelectGroup`, and `RadioGroup` `layout` prop will
-match the value of the `layout` set for the `FilterBarInline` component.
+<Banner
```

Comment:
Hmm, I don't think we have used Banner like this before. I think it's good, for visual differentiation.